File: Luyencode.net/C++/VL15.cpp
#include <iostream>
#include <cmath>
using namespace std;
int gcd(int a, int b) {
	if (b == 0) return a;
	return gcd(b, a % b);
}
int main() {
	int a, b, c;
	cin >> a >> b;
	if (b == 0) {
		cout << "INVALID";
	}
	else if (a % b == 0) {
		cout << int(a/b) << endl;
	} 
	else {
		c = abs(gcd(a, b));
		if (b < 0) {
			cout << -int(a/c) << " " << -int(b/c) << endl;
		}
		else {
			cout << int(a/c) << " " << int(b/c) << endl;
		}
	}
	return 0;
}

File: Luyencode.net/Python/VL15.py
def gcd(a,b):
    if b == 0: return a
    else: return gcd(b,a%b)
a,b = [int(i) for i in input().split()]
if b == 0: print("INVALID")
elif a % b == 0: print(int(a/b))
else:
    c = abs(gcd(a,b))
    if b < 0: print(-a//c,-b//c)
    else: print(a//c,b//c)